tests/ml/test_ml_model_pytest.py

```diff
@@ -330,6 +330,17 @@ def test_learning_to_rank(self, objective, compress_model_definition):
             ],
             reverse=True,
         )
+
+        if (ES_VERSION[0] == 8 and ES_VERSION >= (8, 19)) or (ES_VERSION >= (9, 1)):
```

Comment:
if (ES_VERSION[0] == 8 and ES_VERSION >= (8, 19)) or (ES_VERSION >= (9, 1)):

Should also apply to serverless.
A cluster feature would have been a great option for this.